docs/build-with-powerloom/devnet/getting-started.md

```diff
@@ -76,26 +86,170 @@ Clone the repository against the respective branch (main by default). Open the t
 git clone https://github.com/PowerLoom/deploy.git --single-branch powerloom_deploy --branch devnet && cd powerloom_deploy
 ```
 
-#### Step 2: Fork the Snapshotter Computes and Snapshotter Config Repositories
+#### Step 2: Navigate to the Repository Directory:
+Change to the cloned repository's directory:
+
+```bash    
+    cd powerloom-deploy
+```
+
+#### Step 3: Configure the environment variables
+
+1. In the deploy repo's directory, create a new file named `.env`.
+
+2. Copy the contents of [`env.example`](https://github.com/PowerLoom/deploy/blob/devnet/env.example) to `.env`.
```

Comment:
>In the deploy repo's directory, create a new file named .env.
>Copy the contents of [env.example](https://github.com/PowerLoom/deploy/blob/devnet/env.example) to .env.

Why not just a direct command to `cp env.example .env` and then begin filling it in step 3